source/edge-server/configure.txt

```diff
@@ -386,3 +386,80 @@ server, as well as errors that may have occurred.
             "Item": "truepredicate"
          }
       }
+
+.. _upgrade-edge-server-version:
+
+Upgrade the Edge Server Version
+-------------------------------
+
+As Edge Server evolves, you may want to upgrade your Edge Server version
+to take advantage of new functionality.
+
+Prerequisites
+~~~~~~~~~~~~~
+
+Upgrading the Edge Server version requires the host machine to have 
+`jq <https://jqlang.github.io/jq/>`__ installed. If you do not already
+have ``jq`` installed, running an install command tells you to install 
+``jq``.
+
+.. _edge-server-upgrade-commands:
+
+Upgrade Edge Server Commands
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can upgrade the Edge Server version using one of three commands:
+
+- ``make upgrade-major``: Upgrade to the latest major version of Edge Server.
+- ``make upgrade-minor``: Upgrade to the latest minor version of Edge Server.
+- ``make upgrade-force``: Upgrade to the latest version of Edge Server, 
```

Comment:
@sudssm Do we want customers to use the force feature on their own? I assumed this "ok" to include until we officially release. What do you think?

Reply:
Related to this, I saw that major version upgrades aren't supported yet. I started to add a note about that here, but then decided to leave it out. Since that message is available when you try to run the command, I thought I could leave it to the script to provide that info. That means we won't miss updating it in the docs when you do add that support. But I'm open to adding that detail here if you think it's important to communicate in the docs (or if we want to say anything about why).

Reply:
I dont' have a strong preferences either way, so we can leave this as-is